charts/internal/shoot-system-components/charts/csi-driver-node/templates/clusterrolebinding-csi-driver.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "csi-driver-node.extensionsGroup" . }}:{{ include "csi-driver-node.name" . }}:csi-driver
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "csi-driver-node.extensionsGroup" . }}:{{ include "csi-driver-node.name" . }}:csi-driver
+subjects:
+- kind: ServiceAccount
+  name: csi-driver-node
```

Comment:
The existing `ServiceAccount`s are `csi-driver-node-disk` and `csi-driver-node-file`. This is actually the reason for https://github.com/gardener/gardener-extension-provider-azure/pull/73#discussion_r412061497.